packages/docregistry/src/registry.ts

```diff
@@ -819,10 +819,21 @@ namespace DocumentRegistry {
   }
 
   /**
+   * A widget for a document.
+   */
+  export
+  interface IWidget extends Widget {
```

Comment:
Seems simple enough, but should this base in a more general place like coreutils/apputils. Maybe `IReadyWidget`?